Assets/MRTK/Providers/XRSDK/Profiles/DefaultXRSDKConfigurationProfile.asset

```diff
@@ -53,3 +53,4 @@ MonoBehaviour:
     type: 2}
   useServiceInspectors: 1
   renderDepthBuffer: 0
+  enableVerboseLogging: 0
```

Comment:
I am wondering if this is related to the goal of the PR. Not sure if we should let it go into our codebase if it's irrelevant.

Reply:
I forgot to call this out in the PR description. This is a new field from #8127, and, since this profile was reserialized when I made the data provider change, Unity added it to the serialization with the default value.

This will be okay to check-in, otherwise it'll keep adding itself anytime anybody edits the asset.